src/Contracts/AnalysisTextLibrary.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Whatafix\TextTagger\Contracts;
+
+use Whatafix\TextTagger\Contracts\TextTaggerInterface;
+
+//création de la librairie 
+
+//implementation de l'interface TextTagerInterface en utilisant toute ses méthodes 
+//problème  avec l'implementation de l'interface 
+   
+
+class AnalysisTextLibrary implements TextTaggerInterface
+{
+    //cette méthode analyse le texte en découpant les phrases mot par mot 
+    //en vérifiant que ses mots sont associés dans mon tableau associatif à mes tags
+    //une fois cette analyse faite ma fonction à pour but de resortir les tags associés à ces mots
+    public function getTags(string $data):array
+    { 
+        $tags_Feminin =['Bonjour Madame'];
+
+        $tags_Masculin = ['Bonjour Monsieur'];
+
+        $tags_Enfant = ['Bonjour petit'];
```

Comment:
Tu peux faire des constante comme tu avait fait mais elle doivent se trouver dans la définition de ta class

```suggestion
class AnalysisTextLibrary implements TextTaggerInterface
{
    public const TAG_FEMALE = ['Bonjour Madame'];
    public const TAG_MALE = ['Bonjour Monsieur'];
    public const TAG_CHILD = ['Bonjour petit'];
    
    public function getTags(string $data):array
    { 
```